modular_skyrat/modules/emotes/code/emotes.dm

```diff
@@ -394,9 +387,9 @@
 /datum/emote/living/blep
 	key = "blep"
 	key_third_person = "bleps"
-	message = "bleps their tongue out. Blep."
```

Comment:
why remove this though

Reply:
removing flipping and dabbing is fine, but keep spinning

Reply:
> why remove this though

Just makes it not end with Blep.